seed/ui/src/Components/PeerList.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  import { quintOut } from "svelte/easing";
+  import { crossfade } from "svelte/transition";
+
+  import { online, seen } from "../state.js";
+
+  import Peer from "./Peer.svelte";
+
+  const [send, receive] = crossfade({
+    duration: d => Math.sqrt(d * 360),
+
+    fallback(node, _params) {
+      const style = getComputedStyle(node);
+      const transform = style.transform === "none" ? "" : style.transform;
+
+      return {
+        duration: 1200,
+        easing: quintOut,
+        css: t => `
+          transform: ${transform} scale(${t});
+          opacity: ${t}
+				`,
+      };
+    },
+  });
+</script>
+
+<style>
+  .list {
+    padding-top: 1.5rem;
```

Comment:
```suggestion
    padding: 1.5rem 0;
```
Bottom padding is needed too

Reply:
Had that, but because of the `1.5rem` margin bottom of the peer cards it looked whack, we can polish it later.